klc-cert-manager/controllers/keptnwebhookcontroller/keptnwebhookcertificate_controller.go

```diff
@@ -0,0 +1,94 @@
+package keptnwebhookcontroller
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/keptn/lifecycle-toolkit/cert-manager/eventfilter"
+	"github.com/pkg/errors"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	SuccessDuration            = 3 * time.Hour
+	Webhookconfig              = "klc-mutating-webhook-configuration"
+	secretPostfix              = "-certs"
+	certificatesSecretEmptyErr = "certificates secret is empty"
+	namespace                  = "keptn-lifecycle-toolkit-system"
```

Comment:
should we make the namespace configurable as e.g. an env var?

Reply:
just got the info that supporting the installation of the KLT operator in a different namespace will be a future epic, so I think this is fine for now